src/Platform/Microsoft.Testing.Platform/CommandLine/ActivationMode.cs

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Testing.Platform.CommandLine;
+
+/// <summary>
+/// Specifies the activation mode for command line options that support auto-detection or explicit on/off states.
+/// </summary>
+internal enum ActivationMode
```

Comment:
`OnOffAuto` might have been better, but is internal type so probably no reason to not rename it afterwards.